guides/micronaut-http-client/metadata.json

```diff
@@ -8,7 +8,7 @@
   "apps": [
     {
       "name": "default",
-      "features": ["graalvm", "reactor"]
+      "features": ["reactor", "graalvm"]
```

Comment:
why change the order?

Reply:
So that `graalvm` comes at the end of the create app command, since it is 'optional'.
```
mn create-app example.micronaut.micronautguide \
    --features=reactor,graalvm \
    --build=gradle --lang=java
```
If a developer was creating something similar to the guide and didn't want GraalVM it was easier to remove.

Debated whether or not to move the `--feature` line to the bottom.

Reply:
moving features to the end makes sense